vng_api_common/audittrails/models.py

```diff
@@ -0,0 +1,37 @@
+import uuid
+
+from django.contrib.postgres.fields import JSONField
+from django.core.serializers.json import DjangoJSONEncoder
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from ..descriptors import GegevensGroepType
+from .constants import AuditTrailAction
+
+
+class AuditTrail(models.Model):
+    uuid = models.UUIDField(
+        unique=True, default=uuid.uuid4,
+        help_text="Unieke resource identifier (UUID4)"
+    )
+    bron = models.CharField(max_length=50)
+    actie = models.CharField(
+        choices=AuditTrailAction.choices,
+        max_length=50
+    )
+    actieWeergave = models.CharField(
+        max_length=200,
+        blank=True
+    )
+    resultaat = models.IntegerField()
+    hoofdObject = models.URLField()
```

Comment:
Please add `max_length=1000`. The default is 200 but that's too short, so we manually increase it, like here: https://github.com/VNG-Realisatie/gemma-zaakregistratiecomponent/blob/develop/src/zrc/datamodel/models.py#L122